.gitignore

```diff
@@ -4,7 +4,4 @@
 bin/
 obj/
 
-*.dll
```

Comment:
Can you leave this? If you run commands from readme it pollutes `git status`

Reply:
Yeah, sure.